config_db.go

```diff
@@ -103,7 +103,8 @@ var (
 	hostInitMaxTotalDynamicHostsKey = bsonutil.MustHaveTag(HostInitConfig{}, "MaxTotalDynamicHosts")
 	hostInitS3BaseURLKey            = bsonutil.MustHaveTag(HostInitConfig{}, "S3BaseURL")
 
-	podInitS3BaseURLKey = bsonutil.MustHaveTag(PodInitConfig{}, "S3BaseURL")
+	podInitS3BaseURLKey       = bsonutil.MustHaveTag(PodInitConfig{}, "S3BaseURL")
+	maxParallelPodRequestsKey = bsonutil.MustHaveTag(PodInitConfig{}, "MaxParallelPodRequests")
```

Comment:
Can this be made consistent with the above key to be `podInitMaxParallelPodRequestsKey`?